waargonaut.cabal

```diff
@@ -119,12 +121,17 @@ library
                      , generics-sop >= 0.3.2  && < 4
                      , mmorph
 
-
                      , transformers
                      , bifunctors
                      , contravariant
                      , wl-pprint-annotated
 
+                     , hw-json >= 0.6
+                     , hw-prim
+                     , hw-balancedparens
+                     , hw-rankselect
+                     , hw-bits
```

Comment:
It is definitely worth putting bounds on these before release, but for now it should be ok.